pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -337,6 +339,45 @@ func (w *nfdTopologyUpdater) updateNodeResourceTopology(zoneInfo v1alpha2.ZoneLi
 	return nil
 }
 
+func getCPUCoreAttribute(entry string) (v1alpha2.AttributeInfo, error) {
```

Comment:
nit/suggestion: this now looks like a generic function to read any file and return the content as an attribute. I'd suggest to rename the function as such (and change the var names inside the func generic, too)

```suggestion
func readAttributeFromFile(filepath string) (v1alpha2.AttributeInfo, error) {
```